### PR TITLE
Set --alldeps as defaults in repoquery and add --nodeps option

### PR DESCRIPTION
**Pull request by j-mracek**
It sets --alldeps as default according to yum-dnf compatibility.

It also introduce --nodeps option that represent formal default status of
"dnf repoquery --whatrequires"

### CODE REVIEW

**Comment by j-mracek** [comment]
Jenkins, test this please

**Comment by jsilhan** [comment]
I've told you wrong option name. Please change `--nodeps` to `--exactdeps`. You can also reuse the man description form original repoquery ;)

**Comment by jsilhan** [comment]
Also with these bigger changes, can you document it in advance in release notes so we don;t miss them?

**Comment by j-mracek** [comment]
All changes applied and rebased

File: doc/release_notes.rst [hunk 1]
```diff
@@ -23,6 +23,12 @@ Core DNF Plugins Release Notes
 
 
 ====================
+0.1.21 Release Notes
+====================
+
+Setted --alldeps as defaults in Repoquery plugin and added --exactdeps option.
```
Comment (ignatenkobrain) [inline doc/release_notes.rst]: AFAIK in English there is no such word - `setted`, there is just `set`.

**Comment by mluscon** [comment]
Labeling by dnf-2.0 as this is an incompatible change.

**Pull request closed without merging**